expressEngine/server.js

```diff
@@ -14,29 +15,27 @@ import swaggerDocument from './doc/swagger.json';
 
 // Set up the express app
 const app = express();
-
+config();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
 app.use('/health-check', (req, res, next) => {
   res.status(200).json({
     status: 'ok',
     message: 'Health is good',
-  })
+  });
 });
 
 app.use('/api-docs', SwaggerUi.serve, SwaggerUi.setup(swaggerDocument));
 
 app.use('/api', Route);
 
-const PORT = 5000;
-
 db.open().then(async () => {
   try {
     console.log('Db connected successfully');
 
-    await app.listen(PORT);
-    console.log(`App running on port ${PORT}`);
+    await app.listen(process.env.PORT);
```

Comment:
please use destructuring and remove unnecessary consoles.

Reply:
There is no unnecessary console log, Every Server has a console log to know on which port the server is running

Reply:
process.env is predefined in config of 'dotenv' library

Reply:
As per Coding Guidelines. you should use destructuring to use 'PORT'.

Reply:
const { PORT = 5000 } = process.env;

Also, provide a default port, in case the port is not provided in .env the default port can be used. it will increase the reliability of code.

Reply:
changes added